test/mongooseice/udp/refresh_test.exs

```diff
@@ -9,11 +9,7 @@ defmodule MongooseICE.UDP.RefreshTest do
     alias Jerboa.Format.Body.Attribute
     alias Jerboa.Params
 
-    import Mock, only: [
-      called: 1,
-      with_mock: 3,
-      with_mocks: 2
-    ]
+    import Mock
```

Comment:
There are some magic functions required by `Mock` to be imported while using `Mock.with_mock`. This means, that the "suggestion" from `Mock` documentation to `import Mock` is actually a requirement.